### PR TITLE
PR for Release v1.9.0

### PR DESCRIPTION
**Pull request by laxmikantbpandhare**
**Description of the change:**
This PR is for Release v1.9.0

**Motivation for the change:**
Release v1.9.0

### CODE REVIEW

**Review by varshaprasad96 — CHANGES_REQUESTED** [review]
@laxmikantbpandhare can you also update the olm version [here](https://github.com/operator-framework/operator-sdk/blob/60d25c51fa34b68e02691a0194c783d1aed8fd6d/internal/bindata/olm/versions.go#L20)

**Comment by laxmikantbpandhare** [comment]
> @laxmikantbpandhare can you also update the olm version [here](https://github.com/operator-framework/operator-sdk/blob/60d25c51fa34b68e02691a0194c783d1aed8fd6d/internal/bindata/olm/versions.go#L20)

Modified the OLM version.

File: go.mod [hunk 1]
```diff
@@ -28,8 +28,8 @@ require (
 	github.com/thoas/go-funk v0.8.0
 	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
 	golang.org/x/mod v0.4.2
-	golang.org/x/sys v0.0.0-20210521090106-6ca3eb03dfc2 // indirect
-	golang.org/x/tools v0.1.1
```
Comment (estroz) [inline go.mod:32]: There should be no module changes in this PR.

File: Makefile [hunk 2]
```diff
@@ -42,7 +42,7 @@ generate: build # Generate CLI docs and samples
 	go generate ./...
 
 .PHONY: bindata
-OLM_VERSIONS = 0.16.1 0.17.0 0.18.1
+OLM_VERSIONS = 0.16.1 0.17.0 0.18.2
```
Comment (estroz) [inline Makefile:45]: This must be done in a separate PR prior to a release PR

**Review by estroz — CHANGES_REQUESTED** [review]
Code changes outside of the `IMAGE_VERSION` and generated testdata shouldn't be in this PR.

File: Makefile (hunk 2 above)
Comment (laxmikantbpandhare) [reply Makefile:45]: https://github.com/operator-framework/operator-sdk/pull/4995 - Separate PR for OLM version changes.

File: go.mod (hunk 1 above)
Comment (laxmikantbpandhare) [reply go.mod]: Modified.

File: Makefile (hunk 2 above)
Comment (laxmikantbpandhare) [reply Makefile]: PR merged.

File: internal/bindata/olm/manifests.go [hunk 3]
```diff
@@ -118,7 +118,11 @@ func olmManifests0161OlmYaml() (*asset, error) {
 		return nil, err
 	}
 
+<<<<<<< HEAD
```
Comment (estroz) [inline internal/bindata/olm/manifests.go:121]: You have merge conflicts

**Review by estroz — CHANGES_REQUESTED** [review]

File: internal/bindata/olm/manifests.go (hunk 3 above)
Comment (laxmikantbpandhare) [reply internal/bindata/olm/manifests.go]: Yes, removed it.

**Review by estroz — APPROVED** [review]
/lgtm

**Comment by varshaprasad96** [comment]
merging this forcefully as `https://v1-9-x.sdk.operatorframework.io/` link would still fail in sanity-doc checks.
